db/migrate/20240108184649_create_comments.rb

```diff
@@ -4,7 +4,7 @@ def change
       t.references :user, foreign_key: true
       t.references :post, foreign_key: true
       t.string :text
```

Comment:
- [ ] The text for the comments table should be of the type `text`, not `string`. The `text` type allows for storage of more than `string` type